type.go

```diff
@@ -1290,6 +1463,9 @@ func (t *timeType) useInt32() bool {
 }
 
 func (t *timeType) useInt64() bool {
+	// TODO: Write test covering this change. Not even sure if this is correct.
+	//       Nothing in this struct refers to nanos.
+	//return t.Unit.Micros != nil || t.Unit.Nanos != nil
```

Comment:
I'm not sure I understand why this behavior needed to be changed.

Reply:
Oops. This slipped through from another change I was working through. Removed in d532fa2